common/common_structs/src/locked_token_types.rs

```diff
@@ -40,6 +43,23 @@ pub struct UnlockMilestoneEx {
     pub unlock_percent: u64,
 }
 
+#[derive(
+    ManagedVecItem,
+    TopEncode,
+    TopDecode,
+    PartialEq,
+    TypeAbi,
+    NestedEncode,
+    NestedDecode,
+    Clone,
+    Copy,
+    Debug,
+)]
+pub struct InitialUnlockMilestoneEx {
```

Comment:
There's already a struct for this, named UnlockMilestone. Use that one.

Reply:
Done.